guides/computation-development.md

```diff
@@ -0,0 +1,400 @@
+# Computation Development
+
+_This guide will walk through basic algorithm development in the COINSTAC ecosystem. It is intended for algorithm authors and developers._
+
+## Basic overview of COINSTAC system
+
+COINSTAC is an ecosystem for running decentralized algorithms with many clients. It uses a client-server model: while arbitrary calculations can run on clients and servers, the system’s primary use case is differential privacy, where clients run algorithms with data to ensure their privacy. Clients submit their anonymized results to the server, which amalgamates clients’ data and performs interesting computations, such as averaging, modeling, etc.
+
+COINSTAC’s core is written entirely in JavaScript; writing an algorithm requires writing an entry point in this language.
+ However, the system is robust enough to support algorithms in any language. This guide will explain
+
+It’s important to note that *COINSTAC is alpha software*: its rough around the edges. If you have a question or locate a bug please *[open an issue](https://github.com/MRN-Code/coinstac/issues/new)*.
```

Comment:
Might be able to put this sentence in a notice box (really a table), just formatting flavor so it stands out, but feel free to ignore this suggestion. 

I've done similar here: 
https://github.com/zombieprocess/macbook-provisioning/commit/8b4767210d2d4ea1465e5b528a5f54d7ffff5e53#diff-04c6e90faac2675aa89e2176d2eec7d8R8

See the rendered version here: (assuming you can see this stuff.)
https://github.com/zombieprocess/macbook-provisioning/blob/feature/back2-the-present/README.md